libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -121,7 +121,7 @@ class AppPrompt {
   };
 
   decorate = () => {
-    this.elements.closeIcon = toFragment`<button daa-ll="Close Modal" class="appPrompt-close"></button>`;
+    this.elements.closeIcon = toFragment`<button daa-ll="Close Modal" aria-label="Close" class="appPrompt-close"></button>`;
```

Comment:
Should we instead use the cancel placeholder? When PEP will be available on ROW, we wouldn't want the voice over to announce the English version.

Reply:
Good point! Made the change